h2o-automl/src/test/java/ai/h2o/automl/targetencoding/TargetEncoderMojoWriterTest.java

```diff
@@ -0,0 +1,64 @@
+package ai.h2o.automl.targetencoding;
+
+import hex.ModelStubs;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import water.Scope;
+import water.TestUtil;
+import water.fvec.Frame;
+
+import java.io.File;
+import java.io.FileOutputStream;
+
+import static org.junit.Assert.*;
+
+public class TargetEncoderMojoWriterTest extends TestUtil implements ModelStubs {
+
+  @BeforeClass
+  public static void stall() { stall_till_cloudsize(1); }
+
+  Frame trainFrame = parse_test_file("./smalldata/gbm_test/titanic.csv");
```

Comment:
this should be done in the test itself, otherwise it is leaking data if the test is not executed

Reply:
@michalkurka I see your point. There is a chance that exception will make it to skip removal.